libsol/transaction_printers.c

```diff
@@ -5,6 +5,10 @@
 #include "transaction_printers.h"
 #include "util.h"
 
+#define ADVANCE_INFOS() \
```

Comment:
the referenced variables aren't globals. using a macro like this is unintuitive. remove